app/controllers/volunteers_controller.rb

```diff
@@ -165,10 +165,11 @@ def update
   # switch to a particular user
   def switch_user
     volunteer = Volunteer.find(params[:volunteer_id].to_i)
-    volunteer_region_ids = volunteer.regions.collect{ |region| region.id }
-    admin_region_ids = current_volunteer.assignments.collect do
-       |assignment| assignment.admin ? assignment.region.id : nil
-    end.compact
+    volunteer_region_ids = volunteer.regions.pluck(:id)
+    admin_region_ids =
+      current_volunteer.assignments.collect do |assignment|
+        assignment.admin ? assignment.region.id : nil
+      end.compact
 
```

Comment:
@vidoseaver I actually found this in volunteer.rb: 

```
def admin_region_ids(strict = false)
    admin_regions(strict).pluck(:id)
  end

  def admin_regions(strict = false)
    if super_admin? && !strict
      Region.all
    else
      assignments.
        eager_load(:region).
        where(admin: true).
        collect(&:region)
    end
  end
```

So we could even clean these calls up further.